src/content/blog/whats-new-august-2024.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: "What's new in Astro - August 2024"
+description: "August 2024 - STUFF, STUFF, and more!"
+publishDate: "August 31, 2024"
+authors:
+  - sarah
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-august/og.webp"
+lang: "en"
+---
+import MonthlyShowcaseGrid from './_whats-new-components/MonthlyShowcaseGrid.astro';
+import ShowcaseGridById from './_whats-new-components/ShowcaseGridById.astro';
+import ThemeGrid from './_whats-new-components/ThemeGrid.astro';
+import YouTubeGrid from './_whats-new-components/YouTubeGrid.astro';
+
+Some exciting news from around the Astro ecosystem this month. Let's dive in!
+
+## Updates from Mission Control - UPDATE THESE
+
+- 5.0 alpha
+- official reddit https://www.reddit.com/r/withastro/
+- Astro Cloudinary SDK https://astro.cloudinary.dev/
+
```

Comment:
I think this empty line is messing up rendering:
<img width="873" alt="image" src="https://github.com/user-attachments/assets/cb296fe3-c1e0-46d7-af2c-9bdc87322057">

Instead of being interpreted as two lists, Markdown thinks this is all one list but with paragraphs inside the list items and I guess that messes with out bullet styles…

Simplest fix is probably to remove the empty line or to add some kind of separation (text, heading or whatever)

Reply:
yes, this is still just draft content! No worries!